src/engine/client/client.cpp

```diff
@@ -423,13 +423,13 @@ void CClient::SendInput()
 	SendMsg(&Msg, MSGFLAG_FLUSH);
 }
 
-const char *CClient::LatestVersion()
+const char *CClient::LatestVersion() const
 {
 	return m_aVersionStr;
 }
 
 // TODO: OPT: do this alot smarter!
-int *CClient::GetInput(int Tick)
+int *CClient::GetInput(int Tick) const
```

Comment:
I believe this can't be marked as `const` because it returns a non-`const` pointer into a member of `this`.